massa-async-pool/src/lib.rs

```diff
@@ -16,33 +16,32 @@
 //!
 //! ```json
 //! {
-//!     "sender": "xxxx",  // address that sent the message and spent max_gas*gas_price+coins on emission
+//!     "sender": "xxxx",  // address that sent the message and spent fee + coins on emission
```

Comment:
From what I understand shouldn't this be `max_gas * fee + coins`?

Reply:
Ah ok I get it now it's because it's leveraged in `AsyncMessageId` computation, resolving